webui/react/src/components/TaskList.tsx

```diff
@@ -115,8 +110,8 @@ const TaskList: React.FC<Props> = ({ workspace }: Props) => {
   const { activeSettings, resetSettings, settings, updateSettings } =
     useSettings<Settings>(stgsConfig);
   const { canCreateNSC, canCreateWorkspaceNSC } = usePermissions();
-  const fetchWorkspaces = useEnsureWorkspacesFetched(canceler);
```

Comment:
You removed this effect from the component even though TaskList does require the workspaces to be fetched to function.